src/query-parser/clauses/where-clause.ts

```
@@ -0,0 +1,637 @@
+      throw new Error(
+        `Logical operator '${operator}' needs at least one condition on model '${model.name}'`
+      );
+    }
+
+    switch (operator) {
+      case "AND":
+        return this.adapter.builders.AND(ctx, ...conditionSqls);
+      case "OR":
+        return this.adapter.builders.OR(ctx, ...conditionSqls);
+      case "NOT":
+        const firstCondition = conditionSqls[0];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)